src/data/init/init-moves.ts

```diff
@@ -1463,7 +1473,7 @@ export function initMoves() {
       .bounceable(),
     new AttackMove(MoveId.WAKE_UP_SLAP, ElementalType.FIGHTING, MoveCategory.PHYSICAL, 70, 100, 10, -1, 0, 4)
       .attr(MovePowerMultiplierAttr, (user, target, move) =>
-        targetSleptOrComatoseCondition(user, target, move) ? 2 : 1,
+        targetSleptOrComatoseCondition(user, target, move, true) ? 2 : 1,
```

Comment:
```suggestion
        targetSleptOrComatoseCondition(user, target, move) ? 2 : 1,
```
Huh? How did that happen lol.

Reply:
Oh I see, `targetSleptOrComatoseCondition` wasn't updated.